views/partials/component/container.html

```diff
@@ -0,0 +1,29 @@
+<div class="o-registry-ui__component--container">
+
+
+	{{> component/header}}
+
+	{{#if demos}}
+		{{> component/type/demos}}
+	{{/if}}
+
+	{{#if images}}
+		{{> component/type/images}}
+	{{/if}}
+
+	{{#if service}}
+		{{> component/type/service}}
+	{{/if}}
+
+	<div class="o-registry-ui__component--support">
+		<div class="o-registry-ui__component--contact">
+		<h4>Support</h4>
+			<p><span>EMAIL </span><a href="mailto:{{component.support.email}}?subject={{component.name}}">{{component.support.email}}</a></p>
```

Comment:
Know it's a bit pedantic, but these should be capitalised normally and then uppercased with CSS ideally